app/controllers/notes_controller.rb

```diff
@@ -73,13 +74,14 @@ def destroy
   # Once a valid @node is set by the previous filter we look for the Note we
   # are going to be working with based on the :id passed by the user.
   def find_or_initialize_note
-    if params[:id]
-      @note = @node.notes.find(params[:id])
-    elsif params[:note]
-      @note = @node.notes.new(note_params)
-    else
-      @note = @node.notes.new
-    end
+    @note ||=
```

Comment:
`app/controllers//nested_node_resource_controller.rb` also provides a find_or_initialize method, it would be good it that's updated to the new `||=` convention you're following in this PR

Reply:
Old this memoization, now that `find_or_initialize_X` is no longer being called in the concern, is not necessary, because `find_or_initialize_X` is only going to be called once, as before.
Reverting these changes.